tools/marvin/marvin/lib/common.py

```diff
@@ -1395,3 +1399,199 @@ def isNetworkDeleted(apiclient, networkid, timeout=600):
         time.sleep(60)
     #end while
     return networkDeleted
+
+
+def createChecksum(self, virtual_machine, disk, disk_type):
```

Comment:
Please pass explicit parameters to function than passing self object. It's always good to pass explicit parameters than wrapping the parameters in big object. It also enhances the function definition so that to improve readability of the code.